avm/res/kubernetes/connected-clusters/tests/e2e/waf-aligned/main.test.bicep

```
@@ -0,0 +1,33 @@
+targetScope = 'subscription'
+
+metadata name = 'Deploy connected clusters in WAF aligned configuration'
+metadata description = 'This test deploys a connected clusters.'
```

For consistency with other modules
```suggestion
metadata name = 'WAF-aligned'
metadata description = 'This instance deploys the module in alignment with the best-practices of the Azure Well-Architected Framework.'
```